docs/src/modules/components/WhatsNewLayout.js

```diff
@@ -17,6 +17,102 @@ import TimelineDot from '@mui/lab/TimelineDot';
 import TimelineOppositeContent from '@mui/lab/TimelineOppositeContent';
 
 const entries = [
+  {
+    title: 'MUI X v8',
+    description: 'Cutting-edge features, enhanced customization, and full support for native ESM.',
+    date: new Date(2025, 3, 17),
+    url: 'https://mui.com/blog/mui-x-v8/',
+    highlightList: [
+      {
+        title: 'Data Grid – Pivoting [Premium]',
+        url: 'https://mui.com/blog/mui-x-v8/#pivoting',
+      },
+      {
+        title: 'Data Grid – Ask Your Table (AI Assistance) [Premium]',
+        url: 'https://mui.com/blog/mui-x-v8/#ask-your-table-ai-assistance-in-the-data-grid',
+      },
+      {
+        title: 'Data Grid – Server-Side Aggregation & Lazy Loading [Premium/Pro]',
```

Comment:
For now, we have used sentence casing for previous items. 🤔 
```suggestion
        title: 'Data Grid – Server-side aggregation & Lazy loading [Premium/Pro]',
```